superset/db_engine_specs.py

```diff
@@ -535,6 +535,10 @@ def get_timestamp_column(expression, column_name):
             return f'"{column_name}"'
         return column_name
 
+    @classmethod
+    def convert_dttm(cls, target_type, dttm):
+        return "'{}'".format(dttm.strftime('%Y-%m-%dT%H:%M:%S'))
```

Comment:
wait isn't that identical to the parent's class method?

Reply:
No, this one has the time designator, the parent's class method has whitespace instead. Please see a bit more detailed explanation in the PR description.

Reply:
Any idea as to whether this may be needed at the `PostgresBaseEngineSpec` level?  (would also affect Snowflake, Vertica, Redshift)

Reply:
I'd keep it simple in this PR. It really depends on the way they handle timestamps date/time types. Unfortunately, I am not familiar with either of those databases.